model/pluginconf.go

```diff
@@ -0,0 +1,38 @@
+package model
+
+// PluginConf holds information about the plugin
+type PluginConf struct {
+	ID       uint `gorm:"primary_key;AUTO_INCREMENT;index"`
+	UserID   uint
+	FileName string
+	Token    string `gorm:"unique_index"`
+	Enabled  bool
+	Config   []byte
+}
+
+// PluginConfExternal Model
+//
+// Holds information about a plugin instance for one user.
+//
+// swagger:model PluginConf
+type PluginConfExternal struct {
+	// The user id.
+	//
+	// read only: true
+	// required: true
+	// example: 25
+	UserID uint `json:"user_id"`
+	// The user name. For login.
+	//
+	// required: true
+	// example: P1234
+	Token string `binding:"required" json:"token" query:"token" form:"token"`
+	// The file name of the plugin.
+	//
+	// example: rss.so
+	FileName string `json:"file_name" form:"admin" query:"admin"`
```

Comment:
camelCase is used throughout Gotify so lets not introduce snake_case (:. (Also form and query has weird values)
```suggestion
	FileName string `json:"fileName" form:"fileName" query:"fileName"`
```